src/app/child-dev-project/notes/notes-manager/notes-manager.component.html

```diff
@@ -1,10 +1,15 @@
 <app-entity-list
   [allEntities]="notes"
-  [listConfig]="config"
   [entityConstructor]="entityConstructor"
-  clickMode="none"
-  (elementClick)="showDetails($event)"
+  clickMode="navigate"
```

Comment:
```suggestion
  clickMode="none"
  (elementClick)="showDetails($event)"
```

revert back to original behavior. This is just changed for demonstration/testing currently